src/pal/automation/compile.py

```diff
@@ -19,7 +19,8 @@ def RunCMake(workspace, target, platform):
         print "Running: " + workspace + "/ProjectK/NDP/clr/src/pal/tools/gen-buildsys-clang.sh " + workspace + "/ProjectK/NDP/clr DEBUG"
         print "\n==================================================\n"
         sys.stdout.flush()
-        returncode = subprocess.call(workspace + "/ProjectK/NDP/clr/src/pal/tools/gen-buildsys-clang.sh " + workspace + "/ProjectK/NDP/clr " + target, shell=True)
+        scriptDir = "/ProjectK/NDP/clr/src/pal/tools"
```

Comment:
I'm not so sure about this. Does this need workspace?

Reply:
I believe the files in src/pal/automation are dead. As you can see, they were never touched after the initial release of opensource coreclr. @jkotas I assume these were used when we were still mirroring between the TFS and github, right?

Reply:
Yep, any scripts with ProjectK paths are dead code for sure.

Reply:
Should I remove this file instead?

Reply:
Yes, all files in this directory